src/saml2/config.py

```diff
@@ -569,13 +576,58 @@ def ecp_endpoint(self, ipaddress):
         return None
 
 
+class eIDASConfig(Config):
+    @classmethod
+    def assert_not_declared(cls, error_signal):
+        return (lambda x: x is None,
+                partial(error_signal, message="not be declared"))
+
+    @classmethod
+    def assert_declared(cls, error_signal):
+        return (lambda x: x is not None,
+                partial(error_signal, message="be declared"))
+
+
+class eIDASSPConfig(SPConfig, eIDASConfig):
+    def validate(self):
+        validators = [
+            RuleValidator(
+                "single_logout_service",
+                self._sp_endpoints.get("single_logout_service"),
+                *self.assert_not_declared(should_warning)
```

Comment:
I'm not 100% with this unpacking but I ended up using it to avoid the x2 lines. I'm open to suggestions for alternatives here